src/metabase/upload.clj

```diff
@@ -513,6 +514,16 @@
            (field->db-type driver field->new-type)
            args)))
 
+(defn- invalidate-cached-models! [table]
```

Comment:
I've tested this manually, but I'm not sure where to start with writing an automated test. The model caching code is um not that modular. 🍝

Reply:
IDK, maybe the test should capture call to `invalidate!` with correct model-ids?

Reply:
Yeah I guess a mockist test is better than nothing. They are a bit smell in my book though.

Reply:
Yeah, but at least we catch that _some_ logic is executed - to serve as notification if someone removes/breaks it.

Reply:
Added, and glad I did - it found some issues 😄